ownCloud/Client/Actions/EditDocumentViewController.swift

```diff
@@ -94,6 +94,14 @@ class EditDocumentViewController: QLPreviewController, Themeable {
 		}
 	}
 
+	override func viewDidAppear(_ animated: Bool) {
+		super.viewDidAppear(animated)
+		// Activate editing mode by faking a tap on pencil icon. Unfortunately that's the only way to do it apparently
+		if let items = self.navigationItem.rightBarButtonItems, items.count == 1, let editButton = items.first?.customView as? UIButton {
```

Comment:
On the iPad there are two buttons available in the navigation bar on the right side (Markup and Share). Please add an extra check there, otherwise markup will not be enabled on the iPad.

Reply:
@hosy addressed this issue as well as editing mode not being activated on the device.